ncm-grub/src/main/pan/components/grub/schema.pan

```diff
@@ -11,6 +11,14 @@ include 'pan/types';
  line, where the first field matches the file_user parameter.
 }
 
+type grub_argument = {
+    @{a value for the argument (creates key=value when present; only key is generated when missing
+      (a list is converted in comma-separated list of strings).}
+    "value" ? element with is_string(SELF) || is_long(SELF) || is_list(SELF)
```

Comment:
Should we restrict leading/trailing white-space for string values?

Reply:
dunno but thanks for reminding i need to quote values with empty space in it